sinabs/backend/dynapcnn/utils.py

```diff
@@ -15,6 +15,7 @@
 if TYPE_CHECKING:
     from sinabs.backend.dynapcnn.dynapcnn_network import DynapcnnNetwork
 
+DEFAULT_IGNORED_LAYER_TYPES = (nn.Identity, nn.Dropout, nn.Dropout2d, nn.Flatten)
```

Comment:
For Dropout layers, don't we have to account for parameter rescaling? Is this handled somewhere?

Reply:
Until now dropout layers have been simply ignored at conversion. This PR does not change that. I would argue that this is the correct behavior. Because of the way dropout is implemented in PyTorch, there should be no scaling issue: During training, those outputs that are not dropped are scaled up accordingly. During inference nothing is dropped and nothing is scaled.